### DISCUSSION TITLE
how to access old nodejs versions

### DISCUSSION THREAD

**Discussion by Redo1212** [community/community#67814] [Discussions]
### Select Topic Area

Question

### Body

how to access old nodejs versions

**Comment by Atakan75** [comment]
https://nodejs.org/tr/download/releases

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
